src/Switch.xml

```diff
@@ -23,15 +23,19 @@
                 <enumerationValue key="never">Never</enumerationValue>
             </enumerationValues>
         </property>
-        <property key="label" type="string" required="false" defaultValue="Label">
+        <property key="label" type="translatableString">
             <caption>Label</caption>
             <category>Appearance</category>
             <description/>
+            <translations>
+                <translation lang="en_US"/>
+                <translation lang="nl_NL"/>
```

Comment:
It does not make sense to have an empty translatable string. Please check documentation on this.
It was even crashing the web modeler, does it still?
By removing the default value it should be set in the translations. Both languages could be set with `defaultValue="Label"`